### ISSUE TITLE
A11y_TrydotNetWebsite_Focus Visible: Keyboard focus is not visible when we press 'Tab' after run button.

### ISSUE THREAD

**Issue by shraddhaapatil** [dotnet/try#960]
GitHubTags:#A11ySev2;#A11yTCS;#A11yWCAG2.1;#DesktopWeb;#.NET Websites;#Win10;#P2-TrydotNetWebsite-Aug21;#WCAG2.4.7;#Focus Visible;#FTP;#P2-WCP;#Keyboard

### Environment Details:​
Application Name: TrydotNetWebsite
Standard Violated: WCAG2.4.7 

### Repro Steps:​
1.Hit the URL  https://try.dot.net
2.Tab till Run button.
3.Verify whether focus is visible or not after pressing tab after Run button.

### Actual Result:
Keyboard focus is not visible when we press 'Tab' after run button.

### Expected Result: ​
Keyboard focus should be visible when we press 'Tab' after run button.

### User Impact:
Keyboard users will face difficulty in navigation if Keyboard focus is not visible when we press 'Tab' after run button.
![960_A11y_TrydotNetWebsite_Focus Visible Keyboard focus is not visible when we press 'Tab' after run button](https://user-images.githubusercontent.com/90324695/133959572-351f4bfd-5c6a-4b91-a1ce-a355d1a0ca77.gif)

**Comment by shraddhaapatil** [comment]
https://user-images.githubusercontent.com/90324695/133035240-785e2dc3-87d8-45c9-9df2-92f0437a0b1c.mp4

**Comment by shraddhaapatil** [comment]
![960_A11y_TrydotNetWebsite_Focus Visible Keyboard focus is not visible when we press 'Tab' after run button](https://user-images.githubusercontent.com/90324695/133395445-e0d8c11d-4dbe-4306-a1e0-f8ae6f029118.gif)

**Comment by jonsequitur** [comment]
We typically should not be doing this testing directly against https://try.dot.net, as it's intended to be a component in other services via an IFRAME, and the **Run** button is not rendered by Try .NET in those cases. Also, direct access to try.dot.net without an IFRAME will be disabled in the coming months.

**Comment by jonsequitur** [comment]
Closing this because the Run buttons in the hosting sites do not come from https://try.dot.net/.

**Comment by poojapatil36** [comment]
@jonsequitur The issue is after the run button as focus is not visible on any control after run button.
If still its not own by ( https://try.dot.net/) then please share external team details who owns this.
So that we can route it to correct team, please share all the details of external team.
